src/TapRating.tsx

```diff
@@ -97,11 +97,7 @@ const TapRating: React.FunctionComponent<TapRatingProps> = props => {
   useEffect( () => {
     const { defaultRating } = props;
 
-    if ( defaultRating ) {
-      setPosition( defaultRating );
-    } else {
-      setPosition( 5 );
-    }
+    setPosition( defaultRating );
```

Comment:
Can we do
```suggestion
    if ( defaultRating != null ) {
      setPosition( defaultRating );
    } else {
      setPosition( 5 );
    }
```

Reply:
Yes, this could be a better approach too? @sshivananda. It will check the null defaultRating case and assign 5 if the rating is not found as it seems to me.

Reply:
This will never give 0 as a value to the setPosition in my opinion. This was the approach which I tried before and was not working. @sshivananda

Reply:
Only down-point i find here is that what if the number of stars is less than 5.

Reply:
`if ( defaultRating )` will evaluate to false when defaultRating is 0. `if ( defaultRating != null)` will only evaluate to false when defaultRating is either null or undefined.

Also the README states the if defaultRating is not specified : the default value to be used is 3.

We should either update the code to match readme ( `setPosition( 3 )` ) or update the README.

You bring up a fair point about number of stars being less than the default value - but I think it makes sense to have a default (and note that in a README) in an incremental release.

Making defaultRating a mandatory parameter - is something we can take on in a major version upgrade - if you want to implement that.

Reply:
For now which approach should be suitable? @sshivananda

Reply:
` if ( defaultRating !== null ) ` this has a listing issue @sshivananda `error  Unexpected negated condition  no-negated-condition`

Reply:
For now I would suggest defaulting to 3 - so that its in sync with README. 
After sometime we can make defaultRating mandatory (but that can wait a bit - should not do two major releases in a row)

I think this should work:
```suggestion
    if ( defaultRating === null || defaultRating === undefined ) {
      setPosition( 3 );
    } else {
      setPosition( defaultRating );
    }
```

Reply:
@sshivananda done